feel/feeldiscr/meshstructured.cpp

```diff
@@ -22,11 +22,17 @@
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 #include <feel/feeldiscr/meshstructured.hpp>
+#include <boost/geometry.hpp>
```

Comment:
We should probably add a note on the use of Boost Geometry in Feel++ documentation. this is new definitely